src/core/document.js

```diff
@@ -138,6 +138,15 @@ var Page = (function PageClosure() {
       return shadow(this, 'mediaBox', obj);
     },
 
+    get userUnit() {
+      var obj = this.getPageProp('UserUnit');
+      // Optional according to the spec with a default value of 1.0.
```

Comment:
Can you remove the comment and introduce `DEFAULT_USER_UNIT = 1.0;` constant above?